public/components/freelancer-create.js

```diff
@@ -81,14 +81,16 @@ class CreationForm extends React.Component {
               </label>
             </div>
 
-            <label className="category-selector">
-              Job category:
+            <div className="category-selector">
+              <span className="bar"></span>
+              <label>
+                Job Category
+              </label>
               <select ref="category">
                 <option value="selected" disabled>Please select a job category</option>
```

Comment:
In most forms if you have a "please select a category" then it is not selectable, but it's the default value of the selector. Why have it if it is never shown?

It should show the hint by default, require the user to change it and not allow to reselect the hint.